test/unit/util/test_custom_templates.py

```diff
@@ -0,0 +1,59 @@
+"""Test custom template configuration and rendering."""
+
+from galaxy.util.custom_templates import template
+
+TEMPLATE_RELPATH = "mail/activation-email.txt"
+CONTEXT = {
+    "name": "Jane Doe",
+    "user_email": "jane@example.com",
+    "date": "01-01-2021",
+    "hostname": "example.galaxy.org",
+    "activation_url": "https://example.galaxy.org/fake-activation-link",
+    "terms_url": "https://example.galaxy.org/terms",
+    "contact_email": "contact@example.galaxy.org",
+    "instance_resource_url": "https://example.galaxy.org/resources",
+    "expiry_days": 7,
```

Comment:
```suggestion
```
While these variables are relevant for a real activation message, they are not used in the context of these tests.

Reply:
True!

They are (mostly) used by the default `mail/activation-email.txt` templates, so I thought Jinja2 would throw an undefined error rendering that template if these were omitted.

Apparently that is not the case, it just renders `''` in their place. I'll remove the unnecessary context.

Thanks!